style.css

```
@@ -0,0 +1,344 @@
+    flex-direction: column;
+    justify-content: center;
+}
+
+.promo-mechanics p {
+    font-family: Roboto;
+    font-weight: 400;
+    color: var(--milkapurple);
+}
+
+main {
+    display: flex;
+    flex-direction: column;
+    align-items:center;
+    
+    justify-content: space-between;
+    justify-items: center;
+    /* gap: 1rem; */
+    position: relative;
+    background-color: var(--milkapurple);
+    padding: 2rem;
+}
+
+/* Nu am reusit nicicum sa fie marginea de sus a sectiunii 
+#cow-cutout {
+    display: block;
+    position: relative;
+    top: 0;
+} */
+
+.headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    transform: scale(60%);
+}
+
+main h1, h2 {
+    font-family: Roboto;
+    font-weight: 400;
+    color: #fff;
+    text-align: center;
+}
+
+#product-mood {
+    position: absolute;
+    inset-inline: 0;
+    inset-block-end: 0;
+    width: 70%;
+}
+
+main a {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--highlight-color);
+    color: #fff;
+    border-radius: 2.9rem;
+    font-family: Roboto;
+    font-weight: 400;
+}
+
+.tagline {
+    transform: scale(40%);
+    z-index: 2;
+}
+
+section, nav, .hero-promo {
+    background-color: var(--light-bgr);
+}
+
+section {
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    align-items: center;
+    padding: 2rem;
+    overflow: hidden;
+    background-image: url(/Proiect-HTML-CSS/images/pattern.svg);
+    background-size: cover;
+}
+
+section a {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    justify-items: center;
+    background-color: var(--milkapurple);
+    font-family: Roboto;
+    font-weight: 400;
+    font-size:smaller;
+    color: #fff;
+    border-radius: 1rem;
+}
+
+section p {
+    font-family: Roboto;
+    font-weight: 400;
+    color: var(--milkapurple);
+    text-align: center;
+}
+
+.cocoa {
+    position: absolute;
+    width: 100%;
+    inset-inline-end: -25%;
+}
+
+footer {
+    padding: 2rem;
+    background-color: var(--milkapurple);
+    color: #fff;
+    display: flex;
+   flex-direction: column;
+   align-content: center;
+   justify-items: center;
+   align-items: center;  
+}
+
+h4 {
+    font-family: Roboto;
+    font-size: small;
+}
+
+.menu-footer a {
+    font-family: Roboto;
+    font-weight: 300;
+    color: #fff;
+    font-size: small;
+ }
+
+ .menu-footer {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    align-items: center;  
+ }
+
+.logo-white {
+    max-width: 30vw;
+    display: flex;
+}
+
+.social  {
+transform: scale(40%);
+display: flex;
+flex-direction: row;
+justify-self: center;
+justify-content:space-around;
+}
+
+footer svg {
+    display: flex;
+    flex-direction: row;
+    fill: #f4f4f7;
+}
+
+@media screen and (min-width: 600px) {
+    .hero-promo {
+        display: grid;
+        grid-template-columns: repeat (3, 1fr);

```

În CSS nu se pune spațiu înainte de `(`
```suggestion
        grid-template-columns: repeat(3, 1fr);
```